Environment_Engine/Query/Panels.cs

```diff
@@ -290,5 +290,121 @@ public static List<Panel> PanelsFromPhysical(this List<BH.oM.Physical.Elements.I
 
             return panels;
         }
+
+        [Description("Returns the floor panels of a space represented by Environment Panels")]
+        [Input("panelsAsSpace", "A collection of Environment Panels that represent a closed space")]
+        [Output("floorPanels", "BHoM Environment panel representing the floor of the space")]
+        public static List<Panel> FloorPanels(this List<Panel> panelsAsSpace)
```

Comment:
In hindsight, I think this functionality might be better off within the `Modify` class as that is fundamentally what is being done here, the `PanelType` is being modified based on some parameters, rather than just simply queried.

Reply:
current will be moved